src/app/components/ui/Icon/__tests__/Icon.stories.js

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Icon from '../';
+import { styled } from 'styletron-react';
+
+const icons = [
+  "acciones",
+  "aire",
+  "alerta",
+  "arrow-drop-down",
+  "arrow-drop-up",
+  "arrow_left",
+  "arrow_less",
+  "arrow_more",
+  "arrow_right",
+  "atras",
+  "borrar",
+  "caldera",
+  "calendario",
+  "camara",
+  "cerrar",
+  "cif",
+  "clip",
+  "cuadre-de-caja",
+  "defectos",
+  "direccion",
+  "dto",
+  "editr",
+  "estado-aviso",
+  "factura",
+  "filtro",
+  "firma",
+  "historial",
+  "info",
+  "intervencion",
+  "listado",
+  "lupa",
+  "mail",
+  "mantenimiento",
+  "menu",
+  "movil",
+  "observaciones",
+  "papelera",
+  "perfil",
+  "phone",
+  "piezas",
+  "salir",
+  "servicios",
+  "sort",
+  "toggle-off",
+  "toggle-on",
+  "usuario",
+  "ver"
+  ];
+
+const PaddedContainer = styled('div', ({black}) => {
+  return {
+    display: 'flex',
+    fontSize: '20px',
+    flexWrap: 'wrap',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    background: (black && 'black') || 'inherit',
+    color: (black && 'white') || 'inherit'
+  };
+});
+
+const IconHolder = styled('div', {
+  textAlign: 'center',
+  margin: '20px',
+  width: '100px'
+});
+
+storiesOf('Icon', module)
+  .add('Sizes', () => (
+    <PaddedContainer>
+      {
+        icons.map( (icon, ind) => (
+        <IconHolder key={ind}>
+          <PaddedContainer>
+            <Icon
+              icon={icon}
+              fontSize={'small'}
+              style={{height: 'auto'}}
+            />
+            <Icon
+              icon={icon}
+              fontSize={'default'}
+              style={{height: 'auto'}}
+            />
+            <Icon
+              icon={icon}
+              fontSize={'large'}
+              style={{height: 'auto'}}
+            />
+          </PaddedContainer>
+          <br/>
+          <span>{icon}</span>
```

Comment:
use spaces for objects, and for HTML elements. like
```suggestion
          <span> { icon } </span>
```

Reply:
ACK